shared/macros/ocil.jinja

```diff
@@ -129,6 +143,21 @@ ocil: |
 {{{ ocil_clause_entry_audit_syscall() }}}
 {{%- endmacro %}}
 
+{{#
+    OCIL and OCIL clause for adding a unsuccessful syscall to audit logs
+
+:param syscall: The syscall to audit
+:type syscall: str
+
+#}}
+{{% macro complete_ocil_entry_audit_unsuccessful_syscall(syscall) -%}}
```

Comment:
Where do you plan to use this macro? I don't see it being used in this pull request.

Reply:
Good catch, I move some rule to use this now.